### DISCUSSION TITLE
White screen of Death for IOS devices with React.js App

### DISCUSSION THREAD

**Discussion by Mognoid3r** [vercel/community#2667] [Help] [answered]
### Summary

I deployed a basic landing page site to Vercel through Github. The site appears as expected on Desktop however, when viewing on Iphone or Ipad all I see is a white screen.  I am unable to inspect on Iphone and I have a PC.  I am not getting any errors either.  Why is this only happening for IOS devices? 

Has anyone seen this before?

Please feel free to reference my github to view the code:
https://github.com/Mognoid3r/49ers

Here is the website which is viewable on both desktop and Android:
https://49ers.vercel.app/

![image](https://github.com/vercel/community/assets/48867481/3dceff21-4e9b-4d5e-be06-4a12c5734a8f)

### Example

https://49ers.vercel.app/

### Steps to Reproduce

To replicate this issue you can view https://49ers.vercel.app/ on any IOS device.  Compare what is being rendered to what you see on PC or Android.

**Comment by smaeda-ks** [comment, marked as answer]
@Mognoid3r probably this line: https://github.com/Mognoid3r/49ers/blob/main/src/App.css#L14

This isn't a Vercel issue, though

**Reply by smaeda-ks** [reply]
you also forgot to disable these lines: https://github.com/Mognoid3r/49ers/blob/main/src/components/Join/Join.jsx#L11-L23

**Reply by Mognoid3r** [reply]
Is this a CSS issue in your opinion?

**Reply by smaeda-ks** [reply]
Yep, try removing that line and see what happens. It's just Safari behaving differently than other browsers.

**Reply by Mognoid3r** [reply]
I removed the entire folder and deployed but the site is still not showing on IOS.  I'm tempted to strip down the entire site and add files in one by one to find the bug.
